File: PersianCalendar/src/main/java/com/byagowi/persiancalendar/entities/PersianCalendarEvent.java
package com.byagowi.persiancalendar.entities;

import com.byagowi.persiancalendar.calendar.PersianDate;

public class PersianCalendarEvent extends AbstractEvent<PersianDate> {
    public PersianCalendarEvent(PersianDate date, String title, boolean holiday) {
        this.date = date;
        this.title = title;
        this.holiday = holiday;
    }
}

File: PersianCalendar/src/main/java/com/byagowi/persiancalendar/entities/PersianCalendarEvent.kt
package com.byagowi.persiancalendar.entities

import io.github.persiancalendar.calendar.PersianDate

class PersianCalendarEvent(date: PersianDate, title: String, holiday: Boolean) : AbstractEvent<PersianDate>() {
    init {
        this.date = date
        this.title = title
        this.isHoliday = holiday
    }
}
